kinder/roadmap.md

```diff
@@ -2,17 +2,17 @@
 
 This document outlines some goals, non-goals, and future aspirations for kinder.
 
-kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
+kinder is heavily inspired by [kind](https://github.com/kubernetes-sigs/kind), but, in agreement with the kind team,
+was developed as a separated tool with the goal to explore new kubeadm-related use cases, share issues and solutions, and
+ultimately contribute back new features.
```

Comment:
[1]

Reply:
Uhmm... I don't see the same typo there was in [1] here (o-->of). Am I missing something?

Reply:
i though the same typo was present.